functions/index.js

```diff
@@ -395,15 +395,15 @@ const sendWeeklyDigestEmail = (reportSnapshots) => {
 /**
  * Whenever a new document is added to the reports collection, send a notification email.
  */
-exports.reportAdded = functions.firestore.document(`${REPORTS}/{reportId}`)
+exports.sendEmailOnReportCreation = functions.firestore.document(`${REPORTS}/{reportId}`)
```

Comment:
Are these functions used in the app, or is this used in the console?

Reply:
These are used in the app. There was issue with email reception. So I fixed that and have deployed it. To not affect production, as firebase docs suggested, I added new function(sendEmailOnReportCreation), deployed it and then deleted the old reportAdded function.